.github/workflows/ci.yml

```diff
@@ -144,10 +144,6 @@ jobs:
         if: matrix.test-mode == 'race'
         run: gotestsum --format short-verbose -- ./... -race -parallel=8
 
-      - name: run redis tests
-        if: matrix.test-mode == 'defaults'
-        run: gotestsum --format short-verbose -- -p 1 -run TestRedis -tags redistest ./arbnode/... ./system_tests/... -coverprofile=coverage-redis.txt -covermode=atomic -coverpkg=./...
-
       - name: run challenge tests
```

Comment:
When we support both external and internal redis - the CI should run with external as well.

Reply:
Done.